shared/extension/Configuration.hpp

```diff
@@ -29,93 +30,98 @@
 #include "utility/support/hostname.hpp"
 #include "utility/support/yaml_log_level.hpp"
 
+namespace fs = ::std::filesystem;
```

Comment:
I am generally quite opposed to `using`s on the `std` namespace (and its sub-spaces). And I think in this case the code can be refactored in a way that renders this using quite pointless.

Reply:
I'm ok with this one, std::filesystem and std::chrono can make for every long things. That said I would prefer that the using of namespace is as close to the useage scope as possible